### PR TITLE
Improve formatting of workers CLI

### PR DESCRIPTION
**Pull request by arajasek**
Subsumes #5487

### CODE REVIEW

File: cmd/lotus-miner/sealing.go [hunk 1]
```diff
@@ -91,7 +91,10 @@ var sealingWorkersCmd = &cli.Command{
 
 			var barCols = uint64(64)
 			cpuBars := int(stat.CpuUse * barCols / stat.Info.Resources.CPUs)
-			cpuBar := strings.Repeat("|", cpuBars) + strings.Repeat(" ", int(barCols)-cpuBars)
+			cpuBar := strings.Repeat("|", cpuBars)
+			if int(barCols)-cpuBars >= 0 {
+				cpuBar = cpuBar + strings.Repeat(" ", int(barCols)-cpuBars)
```
Comment (Stebalien) [inline cmd/lotus-miner/sealing.go:96]: nit `cpuBars += ...`

**Review by Stebalien — APPROVED** [review]

File: cmd/lotus-miner/sealing.go (hunk 1 above)
Comment (arajasek) [reply cmd/lotus-miner/sealing.go:96]: we don't know that in many places in lotus so i assumed it was bad go-ing

Comment (arajasek) [reply cmd/lotus-miner/sealing.go:96]: but happy to agree

Comment (arajasek) [reply cmd/lotus-miner/sealing.go:96]:
```suggestion
				cpuBar += strings.Repeat(" ", int(barCols)-cpuBars)
```

**Pull request merged**